cli/runtime/config/context_test.go

```diff
@@ -329,8 +329,9 @@ func TestSetCurrentContext(t *testing.T) {
 			}
 			currSrv, err := GetCurrentServer()
 			assert.NoError(t, err)
-			// server is updated only for K8s Ctx Type, so test-mc remains the current server
-			assert.Equal(t, "test-mc", currSrv.Name)
+			if err != nil {
```

Comment:
Nit: We have a `assert.NoError(t, err)` at line 331, do we still need to check for `err != nil`?

Reply:
Yes. Because `assert.Equal(t, tc.ctxName, currSrv.Name)` test should only be executed when there is no error.
Without this, it throws error because server is not updated.

Reply:
Aha. I had assumed of the assert failed it would stop the test.  Thanks for the explanation.

Reply:
@anujc25 I just realized we should probably have `if err == nil` in this case instead of the opposite, no?  But looking at the goDoc for `assert.NoError()` we could even do:
```
currSrv, err := GetCurrentServer()
if assert.NoError(t, err) {
	assert.Equal(t, tc.ctxName, currSrv.Name)
}
```

Reply:
@marckhouzam I have PR out to address this: https://github.com/vmware-tanzu/tanzu-framework/pull/3432